### ISSUE TITLE
ProductARActivity is not needed

### ISSUE THREAD

**Issue by tiro105** [viromedia/virocore#45]
https://github.com/viromedia/virocore/tree/master/ARRetail/app/src/main/java/com/example/virosample

Hi.
I would like to propose two modifications.
- Delete ProductARActivity
- Rename ProductARActivityComplete to ProductARActivity.

First, ProductARActivity does the same thing as ProductARActivityComplete (INTENT_PRODUCT_KEY also exists in ProductARActivityComplete).
Next, ProductARActivity's name is easily and simply than ProductARActivityComplete.

I fixed it, so please give me permission if necessary.

ViroCore is so cool. Thanks.

**Comment by dthian** [comment]
Hi @tiro105, thanks for the suggestion, ProductARActivityComplete and ProductArActivity are actually both used in conjunction with the [ARRetail tutorial](https://blog.viromedia.com/tutorial-how-to-build-amazons-ar-view-for-arcore-android-using-virocore-and-java-ba1cc3ff2d87). 

Thus, to simplify the code base here, I've created a separate ARRetail project with starting sample code tailored to the tutorial, of which is then compressed and uploaded to the medium blog. 

Finally, I've removed ProduceARActivityComplete in favor of ProductARActivity and have made the changes to this git repo. Ideally, developers can then refer to this final activity should they get lost in the tutorial. 

Theses changes should address your concerns and suggestions as mentioned above, do let me know if you have any questions.
